src/main/java/com/jcertif/pic/sonar/oauth/OAuthPlugin.java

```diff
@@ -67,6 +69,9 @@ public OAuthExtensions(org.sonar.api.config.Settings settings) {
 
         @Override
         public Object provide() {
+            List<Class> myUnusedVar = Lists.newArrayList();
+            String myUnusedStr = null;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "myUnusedStr" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1481)